web_hosting_manager/app/utils/app_utils.js

```diff
@@ -4,6 +4,29 @@ export const trimErrorMsg = (msg) => {
   return msg.slice(index).trim()
 };
 
+export const parseErrorMsg = (err) => {
+  switch(err.code) {
```

Comment:
I think all the messages strings shall be actually loaded here with I18n.t(...) rather than having them hard-coded here.